Ignore more packages for arch-based images

dasJ opened the pull request:
contrib/mkimage-arch.sh now also ignores the packages nano,vi,tar, and
licenses.
These don't belong in a container that is minimal (as arch is).

dasJ committed:
Ignore more packages for arch-based images

contrib/mkimage-arch.sh now also ignores the packages nano,vi,tar,
licenses, device-mapper,netctl,systemd-sysvcompat, and usbutils.
These don't belong in a container that is minimal (as arch is).

Signed-off-by: Janne Heß <jannehess@gmail.com>

jfrazelle commented:
Can you please sign your commits following these rules:

https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

The easiest way to do this is to amend the last commit:

`git commit --amend -s --no-edit && git push -f`

jfrazelle commented:
LGTM

dasJ committed:
Delete arch container after testing

When testing the new image of archlinux (after it was built), it gets
deleted.

Signed-off-by: Janne Heß <jannehess@gmail.com>

dasJ commented:
Ignore even more packages now and delete the test container after test succeeded.

tianon commented:
This is basically a dupe of #9764, right?

dasJ commented:
Yep
